web/src/i18n/locales/en/app.yaml

```diff
@@ -0,0 +1,20 @@
+# These are shared throughout the app.
+
+# This is the title displayed in the sidebar.  First is when logged in, second
```

Comment:
Would it be useful to have comments in all of the files describing which part of the site they apply to, as well as some descriptions of the sections of the file?

Reply:
I think that'd help a lot. Does it make most sense to have it be the first line, above the `title`?